Dshop/apps/products_catalogue/models.py

```diff
@@ -20,9 +20,19 @@ def save(self, *args, **kwargs):
         return super().save(*args, **kwargs)
 
 
+class CeneoCategory(models.Model):
+    name = models.CharField(max_length=200)
+
+    class Meta:
+        verbose_name = 'CeneoCategory'
+        verbose_name_plural = 'CeneoCategories'
+    
+    def __str__(self):
+        return self.name
+
 class Category(CatalogueItemModel):
     parent = models.ForeignKey('self', blank=True, null=True, on_delete=models.CASCADE)
-
+    ceneo_category_id = models.ForeignKey(CeneoCategory, blank=True, null=True, on_delete=models.CASCADE)
```

Comment:
```suggestion
    ceneo_category_id = models.ForeignKey(CeneoCategory, blank=True, null=True, on_delete=models.SET_NULL)
```
I think this would be safer. We don't want to delete category (either side) when one of those is deleted, no?

Reply:
done